File: Chapter04/FramedText/FramedText/FramedText/FramedTextPage.cs
using System;
using Xamarin.Forms;

namespace FramedText
{
    public class FramedTextPage : ContentPage
    {
        public FramedTextPage()
        {
            BackgroundColor = Color.Aqua;

            Content = new Frame
            {
                OutlineColor = Color.Black,
                BackgroundColor = Color.Yellow,
                HorizontalOptions = LayoutOptions.Center,
                VerticalOptions = LayoutOptions.Center,

                Content = new Label
                {
                    Text = "I've been framed!",
                    FontSize = Device.GetNamedSize(NamedSize.Large, typeof(Label)),
                    FontAttributes = FontAttributes.Italic,
                    TextColor = Color.Blue
                }
            };
        }
    }
}

File: Chapter04/FS/FramedText/FramedText/FramedText/FramedTextPage.fs
namespace FramedText

open Xamarin.Forms

type FramedTextPage() = 
    inherit ContentPage(BackgroundColor = Color.Aqua,
                        Content = Frame(OutlineColor = Color.Black,
                                        BackgroundColor = Color.Yellow,
                                        HorizontalOptions = LayoutOptions.Center,
                                        VerticalOptions = LayoutOptions.Center,
                                        Content = Label(Text = "I've been framed!",
                                                        FontSize = Device.GetNamedSize(NamedSize.Large, typeof<Label>),
                                                        FontAttributes = FontAttributes.Italic,
                                                        TextColor = Color.Blue)))



